pwnlib/commandline/template.py

```diff
@@ -1,8 +1,4 @@
-#!/usr/bin/env python2
-from __future__ import absolute_import
-from __future__ import division
```

Comment:
Why?

Reply:
My bad :) I thought since Python 2 is dead that this is unnecessary code :) I just leave it there....